templates/k8s/cakeshop.yaml.erb

```diff
@@ -1,3 +1,11 @@
+<%-
+def set_node_template_vars(values)
```

Comment:
can delete the function definition `set_node_template_vars(values)` here and add `@Service_Prefix` to `qubernetes` https://github.com/ConsenSys/qubernetes/blob/master/qubernetes#L22